src/clj/game/cards/resources.clj

```diff
@@ -1021,7 +1023,12 @@
                  :effect (req (gain-credits state side 1)
                               (trigger-event state side :spent-stealth-credit card)
                               (when (zero? (get-counters (get-card state card) :credit))
-                                (trash state :runner card {:unpreventable true})))}]}
+                                (trash state :runner card {:unpreventable true})))}]
+    ; See Net Mercur for why this implementation was chosen
+    :interactions {:pay-credits {:req (req (and (:run @state)
+                                                (= :ability (:source-type eid))
+                                                (has-subtype? target "Icebreaker")))
```

Comment:
This needs to be just `(:run @state)`, cuz folks like me spend Ghost Runner credits for SMC installs and Hayley triggers.

Reply:
Oh you're right. While Net Mercur is triggering all over the place (and you want to use it for something other than pumping breakers like 1 in 5 times), Ghost Runner is only during the run and it should be fine having it open the prompt for trashing / traces / SMC as well in those cases.